pom.xml

```diff
@@ -92,7 +98,25 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-logging</artifactId>
+			<exclusions>
+				<exclusion>
+					<!-- Exclude logback to prevent multiple SLF4J binding warning -->
+					<groupId>ch.qos.logback</groupId>
+					<artifactId>logback-classic</artifactId>
+				</exclusion>
+				<exclusion>
+					<!-- Exclude logback to prevent multiple SLF4J binding warning -->
+					<groupId>ch.qos.logback</groupId>
+					<artifactId>logback-core</artifactId>
+				</exclusion>
+			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
1. Do you need the various logback-core exclusions if you're defining this here? 
2. Have you done a before and after comparison of the log output to ensure there's no missing log data or unexpected formatting of logs?